### ISSUE TITLE
Select on relationships

### ISSUE THREAD

**Issue by revictus** [sdispater/orator#82]
In Laravel it is possible to do a select on specific columns from relationship, is this currently possible in orator?

Example: http://blog.laravel.in/get-specific-columns-from-laravel-relationships/

**Comment by joaobarbosa** [comment]
@revictus I think it might be possible using eager loading constraints features (but I didn't try it yet). Check it out: https://orator-orm.com/docs/0.8/orm.html#eager-load-constraints

I think you might be able to customize the queries, adding the select statement.

**Comment by sdispater** [comment]
@revictus If I understand it correctly, it would be something like this:

```python
class Company(Model)

    @has_many
    def core_values(self):
        # Relationships method accepts a Builder instance
        return CoreValue.select('company_id', 'title', 'id)
```
If this is the case, that is not currently supported by Orator. When returning a Builder instance in a relationship method, columns are not carried over. However, it is something that might be useful so I will look into it.

**Comment by revictus** [comment]
@sdispater I suspected that it wasn't currently supported as the example you laid out is what I was attempting to accomplish. 

If this could make it into a future update of Orator it would be very helpful

**Comment by sdispater** [comment]
@revictus I just pushed a fix (825e83afc2802c22d626aa92d38e14355c14416b) on the `develop` branch so that columns can be specified in a relationship. You can test it out if you want.

It will be officially available in the next `0.9` release.
